recipes/plutosvg/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conan import ConanFile
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir, rename, replace_in_file
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.meson import Meson, MesonToolchain
+from conan.tools.microsoft import is_msvc
+import os
+
+required_conan_version = ">=2"
+
+class PlutoSVGConan(ConanFile):
+    name = "plutosvg"
+    description = "PlutoSVG is a compact and efficient SVG rendering library written in C"
+    license = "MIT",
+    topics = ("vector", "graphics", "svg")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/sammycage/plutosvg"
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_freetype": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_freetype": True,
+    }
+
+    languages = "C"
+    implements = ["auto_shared_fpic"]
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        # Public symbols and used in headers
+        self.requires("plutovg/1.0.0", transitive_headers=True, transitive_libs=True)
+        if self.options.with_freetype:
+            self.requires("freetype/2.13.2")
+
+    def build_requirements(self):
+        self.tool_requires("meson/[>=1.2.3 <2]")
+        if not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
+            self.tool_requires("pkgconf/[>=2.2 <3]")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        # Seems like upstream places the header in a different folder from where the installed header is
+        replace_in_file(self, os.path.join(self.source_folder, "source", "plutosvg.h"),
+                        "#include <plutovg.h>",
+                        "#include <plutovg/plutovg.h>")
```

Comment:
This seems like an issue with the creation/usage of this library - upstream usages from plutovsg of plutosvg do not match the installed location of plutovg.

I think this warrants reporting before merging, as patching the source here to match Conan output is iffy at best